lib/model/Product.dart

```diff
@@ -102,6 +103,9 @@ class Product extends JsonObject {
       includeIfNull: false)
   Map<OpenFoodFactsLanguage, String>? productNameInLanguages;
 
+  @JsonKey(name: 'generic_name')
+  String? genericName;
+
```

Comment:
What's that?

Reply:
I checked the diffs and this was in the ProductFields but not in the Product, Its in the normal reponses and since its particially already in the code I just added it

Reply:
Fair enough!

Nothing personal, but there are already tons of fields in `Product` (50+) and none of them has comments.
Adding a `String?` called `genericName`, why not, next one is going to be `dynamic justSomething`?

More seriously: we should definitely add comments on the `Product` fields. In another PR.

Reply:
Yes there I agree with you, we should make in any case. In fact, I've been thinking about updating the entire documentation a bit for a while now. 

 I don't know how to approach this, but it would also be good to somehow manage to have the products in just one place and not in two files in two places each.